core/classes/Templates/TwigTemplateEngine.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * Twig template engine.
+ *
+ * @author Samerton
+ * @license MIT
+ * @version 2.2.0
+ */
+
+use Twig\Environment;
+use Twig\Extension\SandboxExtension;
+use Twig\Loader\FilesystemLoader;
+use Twig\Profiler\Profile;
+use Twig\Sandbox\SecurityPolicy;
+
+class TwigTemplateEngine extends TemplateEngine
+{
+    private Environment $_twig;
+
+    /**
+     * @param string $template Template name to load.
+     */
+    public function __construct(string $template)
+    {
+        $loader = new FilesystemLoader(ROOT_PATH . '/custom/templates/' . $template);
```

Comment:
Maybe also allow path here if people wanna make panel templates or templates for other module releated dashboards in Twig???